planning/behavior_velocity_planner/src/scene_module/blind_spot/scene.cpp

```diff
@@ -309,6 +311,12 @@ int BlindSpotModule::insertPoint(
     // copy from previous point
     inserted_point = inout_path->points.at(std::max(insert_idx - 1, 0));
     inserted_point.point.pose = path_ip.points[insert_idx_ip].point.pose;
+    //! avoid to insert duplicated point
+    if (
+      planning_utils::calcDist2d(inserted_point, inout_path->points.at(insert_idx).point) < 0.001) {
+      inout_path->points.at(insert_idx).point.longitudinal_velocity_mps = 0.0;
+      return insert_idx;
+    }
```

Comment:
Is the check unnecessary if this is used insertVelocity in utils?

Reply:
because blindspot module function has int *stop_index variables and there is no smart way using utils instead for small change